.github/workflows/update_website.yml

```diff
@@ -2,18 +2,14 @@ name: Update Group Website
 
 on:
   workflow_dispatch:
-  workflow_run:
-     workflows: ["Check Webhook Dispatch"]
-     types: 
-       - completed
+  repository_dispatch:
+    types: [repo-push]
```

Comment:
do you need square brackets?

Reply:
Yes. I checked it here:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch

Reply:
nice!